datasets/yelp_polarity/README.md

```diff
@@ -3,6 +3,55 @@ languages:
 - en
 paperswithcode_id: null
 pretty_name: YelpPolarity
+train-eval-index:
+- config: plain_text
+  task: text-classification
+  task_id: binary_classification
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    text: text
+    label: target
+  metrics:
+    - type: accuracy
+      name: Accuracy
+    - type: f1
+      name: F1 macro
```

Comment:
Same comment here as in earlier PRs about using just binary F1 score for these type of datasets